book/controller.rst

```diff
@@ -786,6 +786,20 @@ Just like when creating a controller for a route, the order of the arguments of
 order of the arguments, Symfony will still pass the correct value to each
 variable.
 
+Checking the Validity of a CSRF Token
+-------------------------------------
+
+Sometimes you want to use CSRF protection in an action where you don't want to use a
+Symfony form.
+
+If, for example, you're doing a DELETE action, you can use the
+:method:`Symfony\\Component\\Form\\Extension\\Csrf\\CsrfProvider\\CsrfProviderAdapter::isTokenValid`
+method to check the CSRF token::
```

Comment:
I feel that this sentence should be worded more "smoothy". Though I'm not sure how. Maybe our native speaker @weaverryan has a suggestion.